packages/squiggle-lang/src/fr/plot.ts

```diff
@@ -107,13 +110,28 @@ const _assertYScaleNotDateScale = (yScale: Scale | null) => {
   }
 };
 
+function formatXPoints(
+  xPoints: readonly number[] | null,
+  xScale: Scale | null
+) {
+  const [min, max] = [xScale?.min, xScale?.max];
+  let points = xPoints ? sort(uniq(xPoints)) : null;
+  if (points && min) {
```

Comment:
`min` check should be `min !== undefined` (not sure if `null` is also possible); same with `max`

Reply:
good point, fixed!